### PR TITLE
Use app only auth for request and access token

### LINKED ISSUE #8: Why require accessToken?
This library helps do Twitter Sign In, which should be done with just application credentials, not user credentials. Why does it require `accessToken` and `accessTokenSecret`.

I saw you're using `twit` which has an option `app_only_auth: true` to authenticate only the app. Any reason that I missed?

### PR DESCRIPTION
**Pull request by phuctm97**
Fixes #8

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -137,8 +136,7 @@ module.exports = ({consumerKey, consumerSecret, accessToken, accessTokenSecret,}
         const t = new Twit({
             consumer_key: consumerKey,
             consumer_secret: consumerSecret,
-            access_token: requestToken,
```
Comment (shalvah) [inline index.js:140]: This one should be unchanged. After authenticating as a user, you need the request token and request token secret to retrieve their access token.

File: index.js [hunk 2]
```diff
@@ -32,8 +32,7 @@ module.exports = ({consumerKey, consumerSecret, accessToken, accessTokenSecret,}
         const t = new Twit({
             consumer_key: consumerKey,
             consumer_secret: consumerSecret,
-            access_token: accessToken,
-            access_token_secret: accessTokenSecret,
+            app_only_auth: true
```
Comment (shalvah) [inline index.js:35]: I'll need to test this out first. I know that user and app contexts have different rate limits. Need to be sure there aren't any side effects for auth.

**Comment by phuctm97** [comment]
Just tested this, looks like a user token is required. It's weird.

**Comment by phuctm97** [comment]
Should I close this now or you want to explore more?

**Comment by shalvah** [comment]
Closing 👍

**Pull request closed without merging**